src/test/resources/unit/rpm-systemd-maven-plugin/postinst/postinst-additional-directories-without-user-and-group-expected

```diff
@@ -0,0 +1,12 @@
+
+test -f /environmentfile || echo 'JAVA_OPTS=""' > /environmentfile
+getent group some-group || groupadd -r some-group >/dev/null 2>&1
+getent passwd makubi || useradd -s /bin/false -d /path/to/workingDirectory -g some-group -r -m makubi >/dev/null 2>&1
+
+test -d /path/to/workingDirectory || mkdir /path/to/workingDirectory >/dev/null 2>&1
+test -d /path/to/workingDirectory && chown -R makubi:some-group /path/to/workingDirectory >/dev/null 2>&1
+
+/usr/bin/systemctl preset some.service >/dev/null 2>&1
```

Comment:
Maybe systemctl will not always be in /usr/bin? (sbin?/bin/? in other distributions)? Possible to use $PATH  resolution here?